### ISSUE TITLE
when the lockwindow dissmissed, appwindow is wrongly assigned to be keywindow.

### ISSUE THREAD

**Issue by 84798012** [smartdevicelink/sdl_ios#1501]
when car from running mode with app- lockwindow to non-running mode (dissmiss lockwindow).
Our app's hide window(secure window) is assigned to be keywindow wrongly, and my app is blocked by the secure window, can do nothing.
Here is link for this demo.
https://github.com/84798012/SDL_Window_Rotate_Bug_Reproduce_Demo/tree/master/dissmissblock

**Comment by 84798012** [comment]
seems this issue been fixed, thanks!

**Comment by 84798012** [comment]
fixed, so close it.

**Comment by joeljfischer** [comment]
Hello @84798012, the issues are not resolved until they are merged into a release branch. Please allow us to handle the closing of issues.

**Comment by joeljfischer** [comment]
@84798012 Could you please re-test this bug using PR #1509? We ended up doing a large refactor of the lock screen code and would like to confirm that it still fixes all of your issues. Thank you!

**Comment by 84798012** [comment]
I have test on 1509, the issue fixed. Thanks!
